CMakeLists.txt

```diff
@@ -74,10 +74,14 @@ find_library(
   PATHS ${PROJECT_BINARY_DIR}/open62541 ${PROJECT_BINARY_DIR}/open62541/build ${PROJECT_BINARY_DIR}/open62541/Release/ ${PROJECT_BINARY_DIR}/open62541/Debug/ NO_DEFAULT_PATH
 )
 message(STATUS "OPEN62541_SDK_LIB [${OPEN62541_SDK_LIB}]")
-
+  
 if(NOT STANDALONE_BUILD)
   add_library ( open62541-compat OBJECT ${SRCS} )
+  add_custom_target( quasar_opcua_backend_is_ready DEPENDS open62541-compat )
```

Comment:
We moved the build stage of open62541 into build stage of a quasar server rather than into its CMake configuration stage. Since open62541 dependencies are not trivial (i.e. we use an amalgamate header open62541.h which is generated at build time) CMake needs to know that in order to build e.g. server's AddressSpace, this amalgamation phase must be finished. That's why quasar 1.3.0's master CMakeLists will put a dependency on this newly introduced target, and open62541 will be built BEFORE any server module will be started to get built.

Please let me know if that answers the question and whether it sounds reasonable.

Reply:
Ah, I see - so there is a (pending?) quasar change to hook into this ```quasar_opcua_backend_is_ready```.

Sounds fine to me.